homework-hello-world/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+#![no_std]
+use gstd::{msg, debug};
+use gstd::prelude::*; // String
+
+#[no_mangle]
+pub unsafe extern "C" fn handle() {
```

Comment:
Please avoid to use `unsafe`
```suggestion
pub extern "C" fn handle() {
```